packages/itwinui-css/src/button/button.scss

```diff
@@ -1,33 +1,31 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
-@import '../style/index';
-
-$iui-button-padding: $iui-xs * 4;
-$iui-button-padding-small: $iui-xs * 2;
-$iui-button-padding-large: $iui-xs * 6;
 
 @mixin iui-button {
   @include iui-reset;
   font-family: inherit;
   display: inline-flex;
+  flex-shrink: 0;
   align-items: center;
   vertical-align: middle;
   justify-content: center;
   position: relative;
-  border-radius: $iui-border-radius;
-  line-height: $iui-line-height;
-  font-size: $iui-font-size;
-  font-weight: $iui-font-weight-normal;
+  border-radius: var(--iui-border-radius-1);
+  line-height: var(--iui-size-l);
+  font-size: var(--iui-font-size-1);
+  font-weight: var(--iui-font-weight-normal);
   user-select: none;
   cursor: pointer;
   white-space: nowrap;
   border: 1px solid var(--_iui-button-border-color);
   background: var(--_iui-button-background-color);
   color: var(--_iui-button-text-color);
   gap: var(--_iui-button-gap);
-  height: var(--_iui-button-height);
+  min-height: var(--_iui-button-min-height);
+  min-width: var(--_iui-button-min-height);
```

Comment:
this makes the icon-button square (same as #712)

Reply:
Shouldnt it make it square with just having set icon size + padding...?
And I am not sure if `min-height` is better than `height` ? doesnt it allow the button to be larger than we expect it to be then?

Reply:
setting just one expected value sounds a lot simpler than relying on icon size + padding

also, `min-height` makes sense because we want the height to increase if the content inside is larger for whatever reason. Setting fixed `height` should generally be avoided